yuansfer/dto/recurring/PayPal_BillingCycle_Amount.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+class PayPalBillingCycleAmount(dict):
+
+    def __init__(self, value: int = None, currency_code = None):
+        self.value = value
+        self.currency_code = currency_code
+        self.__dict__ = self
```

Comment:
I don't think this line is needed, why you put it in here?

```
self.__dict__ = self

```

Reply:
This line is to convert object class into dictionary, without this line, i will need to use __dict__ when initializing a new object class

Reply:
@xuehao712  How about add a class method for it? like `to_dict()` ?

Reply:
can you let me know why it is a preferred way than using self.__dict__ to make class object itself a dict?

Reply:
We shouldn't try to change the class structure.  It is fine to create a separate method to convert it to `to_dict()`

Reply:
sure, i will use this method